test/internal/laws/Chain.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var compose = require('./compose');
+var flatMap = require('./flatMap');
+var forall = require('./forall');
+
+
+module.exports = function(equals) {
+  return forall({
+
+    //  m >>= f >>= g = m >>= (f >=> g)
```

Comment:
yes, in [hackage](https://hackage.haskell.org/package/base-4.9.0.0/docs/Control-Monad.html#t:Monad) it's also written with parens